src/lib/containers/StorybookComponentWrapper.tsx

```diff
@@ -67,6 +67,7 @@ export function StorybookComponentWrapper(props: {
   useEffect(() => {
     async function resetCache() {
       await storybookQueryClient.cancelQueries()
+      storybookQueryClient.removeQueries()
```

Comment:
Ensures data is removed from the cache before switching stories. You could end up in a scenario where prod data gets into your mock story otherwise.